azure-ipam/ipam.go

```diff
@@ -9,13 +9,19 @@ import (
 	"github.com/Azure/azure-container-networking/azure-ipam/internal/buildinfo"
 	"github.com/Azure/azure-container-networking/azure-ipam/ipconfig"
 	"github.com/Azure/azure-container-networking/cns"
+	cnscli "github.com/Azure/azure-container-networking/cns/client"
+	"github.com/Azure/azure-container-networking/cns/fsnotify"
```

Comment:
can we move fsnotify package outside of cns directory?

Reply:
fsnotify is in this directory so it can act as a library to be used in CNS